src/cinema/presentation/CinemaController.java

```diff
@@ -0,0 +1,61 @@
+package cinema.presentation;
+
+import cinema.businesslayer.CinemaService;
+import cinema.domainmodel.Seat;
+import cinema.domainmodel.Token;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.server.ResponseStatusException;
+
+import java.util.Map;
+import java.util.Optional;
+
+@RestController
+public class CinemaController {
+
+    @Autowired
+    CinemaService cinemaService;
```

Comment:
Next time, try not to use field injection 
https://blog.marcnuri.com/field-injection-is-not-recommended